python/pyarrow/_flight.pyx

```diff
@@ -856,9 +856,13 @@ cdef class FlightInfo(_Weakrefable):
         endpoints : list of FlightEndpoint
             a list of endpoints where this flight is available.
         total_records : int
-            the total records in this flight, or -1 if unknown
+            the total records in this flight, or -1 if unknown.
         total_bytes : int
-            the total bytes in this flight, or -1 if unknown
+            the total bytes in this flight, or -1 if unknown.
+        ordered : boolean
+            Whether endpoints are in the same order as the data.
+        app_metadata : str
```

Comment:
```suggestion
        app_metadata : bytes
```

Reply:
I guess if we wanted to be a bit more flexible we could allow bytes or str, and call `tobytes` on this.